src/System.Private.Xml/src/System/Xml/Serialization/IXmlSerializable.cs

```diff
@@ -2,7 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if XMLSERIALIZERGENERATOR
+using System.Xml;
```

Comment:
Let's put this outside `#if XMLSERIALIZERGENERATOR` to reduce the code being forked.